### DISCUSSION TITLE
Raspberry Pi Docker

### DISCUSSION THREAD

**Discussion by gvcandyman** [cpvalente/ontime#437] [Q&A]
I am getting a command not found message when I try to send the command docker-compose up -d. I've tried different versions and searched for an answer and can't seem to find a way to start ontime on my Pi. Can someone please help? Thanks

**Comment by cpvalente** [comment]
Hi, thank you for reaching out.
Can you please provide some more context to your problem? 

- What were your installation steps, did you follow the [guide in the docs](https://ontime.gitbook.io/v2/additional-notes/use-in-raspberry-pi)?
- Which pi are you using?
- Which version of docker?

Our docs hint at a command to install docker but fail to provide it. This is an oversight from my part and I will clarify the text

**Reply by gvcandyman** [reply]
i have run curl -sSL https://get.docker.com | sh
i am using a Raspberry Pi 3 Model B Plus Rev 1.3 running 6.1.21-v7+
Docker is 24.0.2

**Comment by gvcandyman** [comment]
this is what i receive when i run the docker-compose up -d command 
bash: docker-compose: command not found

**Reply by cpvalente** [reply]
Are you able to install any other docker images?
This sounds like might be an issue outside ontime?

I haven't been able to recreate but will try and improve documentation

**Reply by gvcandyman** [reply]
do you have an image you recommend? I might be having trouble composing in docker.

**Reply by cpvalente** [reply]
I would recommend trying out hello-world
https://hub.docker.com/_/hello-world/

Can you please link to your docker installation steps along with which OS do you have?

**Comment by gvcandyman** [comment]
I figured out that the scripting had been changed in docker and so the syntax needed to be updated to make this work
